CROSS_CHAIN_RECOVERY.md

```diff
@@ -0,0 +1,35 @@
+# Cross Chain Recovery Instructions
+
+In a cross-chain transaction, you attempt to send funds to an address, but accidentally perform the transaction on the wrong chain.
+
+For example, an exchange may ask you to deposit Bitcoin Cash to the Bitcoin Cash address 36jfYqR3J29nAEfYqgksVJ8iVrvLmEb51v. If you sent Bitcoin to the address by mistake, this would be a BTC->BCH cross-chain transaction.
+
+Cross-chain transaction among BTC, LTC, and BCH can be recovered
+
+## What happens in a cross-chain recovery?
+
+Using the same BTC→BCH scenario as above, the goal is to use the private keys of the Bitcoin Cash wallet (that the user was trying to deposit into) to create a transaction on the Bitcoin Blockchain so that the funds can be recovered. Here's how it works:
+
+1. You are asked to provide the Source Coin (the actual coin that was sent, in this case, BTC).
+   
+2. You are asked to provide the Recovery/Destination Coin (the coin of the wallet that you were trying to deposit into, in this case, BCH).
+   
+3. You are asked to provide the Wallet ID of the wallet that you tried to deposit into. (BCH), as well as the password or private key of the wallet.
+   
+4. You are asked to provide the Transaction ID of the cross-chain transaction (when the user sent the BTC)
+   
+5. You are asked to provide the Destination Address, a valid Source Coin (BTC) address to send the recovered funds to.
+   
+6. The SDK determines the amount of Source Coin (BTC) that can be recovered, and builds a transaction to send that amount to the Destination Address.
+   
+7. The SDK signs this transaction with your private key of the BCH wallet, then gives you the half-signed transaction.
+   
+8. You need to send the half-signed transaction to our support team, who co-signs it with the BitGo key of the wallet.
+   
+9. This completed transaction is a valid (BTC) transaction, and because the transaction is signed with the keys belonging to the BCH wallet, that transaction is allowed to spend from the same address.
+
+## Important Notes
+
+If you are trying to recover the funds from a cold wallet, then you can uncheck the 'Sign Transaction' Checkbox. On unchecking it, you will get the unsigned transaction. You need to sign this transaction with the user private key using the BitGoJS SDK and send the half signed transaction to our support team.
```

Comment:
I believe they can use OVC to sign the transaction as well, not just the BiGoJS SDK. Could you please confirm and update accordingly?

Reply:
yes, cross-chain recovery can be signed using OVC. I checked with BTC->BCH recovery. This is the tx, https://live.blockcypher.com/btc-testnet/tx/f501540fd8e7625ab5a0636626db6b5808f54e994a5cb8ad5d66d4c6f4c40060/

Reply:
thank you for confirming! could you please add the following,
"You need to sign this transaction with the user private key using the BitGoJS SDK or using OVC and send the half signed transaction to our support team."

Reply:
I have already updated that [here](https://github.com/BitGo/wallet-recovery-wizard/pull/135/files#:~:text=You%20need%20to%20sign%20this%20transaction%20with%20the%20user%20private%20key%20using%20the%20BitGoJS%20SDK%20or%20OVC%20and%20send%20the%20half%20signed%20transaction%20to%20our%20support%20team.)

Reply:
sorry I missed it